### ISSUE TITLE
Unable to find step on Python with raw strings.

### ISSUE THREAD

**Issue by derrix060** [alexkrechik/VSCucumberAutoComplete#170]
If in my function I use raw strings, the cucumberautocomplete doesn't work.

Example
```python
@When(r"(?:I |)(tap on|press|click on) the '(.+)' (button|icon)", regexp=True)
def step(context, action, button_name, button_type, count):
    ...
```

```feature
...
When I press the 'Home' button
...
```

If I change my code to:

```python
@When("(?:I |)(tap on|press|click on) the '(.+)' (button|icon)", regexp=True)
def step(context, action, button_name, button_type, count):
    ...
```

it will work.

Is there anything that I can do?

**Comment by alexkrechik** [comment]
Unfortunately I don't see any immediate solution for now, due to current format looks wrong for the most programming languages. Everything that could be done in the future - adding an ability to provide some custom regexp's for the steps getter.

**Comment by alexkrechik** [comment]
Issues could be fixed by using of custom parameters in 2.9.0. Try something like this:
```
    "cucumberautocomplete.customParameters": [
        {
            "parameter":"(r\"(",
            "value":"(\"("
        }
    ],
```

**Comment by derrix060** [comment]
It's working now. I've change it to

```
"cucumberautocomplete.customParameters": [
        {
            "parameter": "r\"",
            "value": "\""
        }
    ],
```
